modules/integration/tests-integration/bpel/src/test/java/org/wso2/bps/integration/tests/bpel/security/SecurityWithServiceDescriptorTest.java

```diff
@@ -78,42 +77,46 @@ public void setEnvironment() throws Exception {
 		requestSender = new RequestSender();
 	}
 
-	@BeforeClass(alwaysRun = true)
-	public void deployArtifact()
-			throws Exception{
+	@BeforeClass(alwaysRun = true) public void deployArtifact() throws Exception {
 		setEnvironment();
 		uploadBpelForTest("SecuredWithServiceDescriptorProcess");
 	}
 
-	@Test(groups = {"wso2.bps", "wso2.bps.security"}, description = "BPEL security test scenario - secure BPEL process with service.xml file")
-	public void securityWithServiceDescriptorTest() throws Exception {
+	@Test(groups = { "wso2.bps",
+	                 "wso2.bps.security" }, description = "BPEL security test scenario - secure BPEL process with service.xml file") public void securityWithServiceDescriptorTest()
+			throws Exception {
 		requestSender.waitForProcessDeployment(backEndUrl + "SWSDPService");
 		ProductConstant.init();
 
-		String securityPolicyPath = ProductConstant.SYSTEM_TEST_RESOURCE_LOCATION + BPSTestConstants.DIR_ARTIFACTS
-		                            + File.separator + BPSTestConstants.DIR_POLICY + File.separator + "utpolicy.xml";
+		String securityPolicyPath =
+				ProductConstant.SYSTEM_TEST_RESOURCE_LOCATION + BPSTestConstants.DIR_ARTIFACTS +
+				File.separator + BPSTestConstants.DIR_POLICY + File.separator + "utpolicy.xml";
 
 		String endpointHttpS = "https://localhost:9743/services/SWSDPService";
 
-		String trustStore = CarbonUtils.getCarbonHome() + File.separator + "repository" + File.separator + "resources" +
-		                    File.separator + "security" + File.separator + "wso2carbon.jks";
+		String trustStore =
+				CarbonUtils.getCarbonHome() + File.separator + "repository" + File.separator +
+				"resources" +
+				File.separator + "security" + File.separator + "wso2carbon.jks";
 		String clientKey = trustStore;
 		OMElement result;
 
 		System.setProperty("javax.net.ssl.trustStore", trustStore);
 		System.setProperty("javax.net.ssl.trustStorePassword", "wso2carbon");
 
-		System.out.println(CarbonUtils.getCarbonHome());
-		ConfigurationContext ctx = ConfigurationContextFactory.createConfigurationContextFromFileSystem(
-								   CarbonUtils.getCarbonHome() + File.separator + "repository" +
-								   File.separator + "deployment" + File.separator + "client", null);
+		if (log.isDebugEnabled()) {
+			log.debug(CarbonUtils.getCarbonHome());
```

Comment:
Instead directly logging, include some meaningful context to the log message. For example log.debug("Carbon Home: " + CarbonUtils.getCarbonHome());